resource_customizations/apps/Deployment/actions/testdata/deployment-pause.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    deployment.kubernetes.io/revision: "1"
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"apps/v1","kind":"Deployment","metadata":{"annotations":{"deployment.kubernetes.io/revision":"1"},"creationTimestamp":"2019-09-12T01:33:53Z","generation":1,"name":"nginx-deploy","namespace":"default","resourceVersion":"6897444","selfLink":"/apis/apps/v1/namespaces/default/deployments/nginx-deploy","uid":"61689d6d-d4fd-11e9-9e69-42010aa8005f"},"spec":{"progressDeadlineSeconds":600,"replicas":3,"revisionHistoryLimit":10,"selector":{"matchLabels":{"app":"nginx"}},"strategy":{"rollingUpdate":{"maxSurge":"25%","maxUnavailable":"25%"},"type":"RollingUpdate"},"template":{"metadata":{"labels":{"app":"nginx"}},"spec":{"containers":[{"image":"nginx:latest","imagePullPolicy":"Always","name":"nginx","resources":{},"terminationMessagePath":"/dev/termination-log","terminationMessagePolicy":"File"}],"dnsPolicy":"ClusterFirst","restartPolicy":"Always","schedulerName":"default-scheduler","securityContext":{},"terminationGracePeriodSeconds":30}}},"status":{"availableReplicas":2,"conditions":[{"lastTransitionTime":"2019-09-12T01:33:53Z","lastUpdateTime":"2019-09-12T01:33:53Z","message":"Deployment does not have minimum availability.","reason":"MinimumReplicasUnavailable","status":"False","type":"Available"},{"lastTransitionTime":"2019-09-12T01:33:53Z","lastUpdateTime":"2019-09-12T01:34:05Z","message":"ReplicaSet \"nginx-deploy-9cb4784bd\" is progressing.","reason":"ReplicaSetUpdated","status":"True","type":"Progressing"}],"observedGeneration":1,"readyReplicas":2,"replicas":3,"unavailableReplicas":1,"updatedReplicas":3}}
+  creationTimestamp: "2021-09-21T22:35:20Z"
+  generation: 2
+  managedFields:
```

Comment:
Could we remove managedFields and kubectl.kubernetes.io/last-applied-configuration from the test yaml?

Reply:
Thanks for the review @jessesuen. I have made the required changes.